docs/reference/NuGet-Client-SDK.md

```diff
@@ -10,9 +10,30 @@ ms.topic: conceptual
 # NuGet Client SDK
 
 The *NuGet Client SDK* refers to a group of NuGet packages:
-
-* [`NuGet.Protocol`](https://www.nuget.org/packages/NuGet.Protocol) - Used to interact with HTTP and file-based NuGet feeds
-* [`NuGet.Packaging`](https://www.nuget.org/packages/NuGet.Packaging) - Used to interact with NuGet packages. `NuGet.Protocol` depends on this package
+* [`NuGet.CommandLine`](https://www.nuget.org/packages/NuGet.CommandLine) - NuGet Command Line Interface.
+* [`NuGet.Indexing`](https://www.nuget.org/packages/NuGet.Indexing) - NuGet's indexing library for the Visual Studio client search functionality.
+* [`NuGet.VisualStudio.Contracts`](https://www.nuget.org/packages/NuGet.VisualStudio.Contracts) - This package contains RPC contracts for NuGet’s Visual Studio Service Broker extensibility APIs.
+* [`NuGet.VisualStudio`](https://www.nuget.org/packages/NuGet.VisualStudio) - This package provides APIs for invoking NuGet services in Visual Studio.
+* [`Microsoft.Build.NuGetSdkResolver`](https://www.nuget.org/packages/Microsoft.Build.NuGetSdkResolver) - MSBuild SDK resolver for NuGet packages.
+* [`NuGet.Build.Tasks.Console`](https://www.nuget.org/packages/NuGet.Build.Tasks.Console) - NuGet Build tasks for MSBuild and dotnet restore.
+* [`NuGet.Build.Tasks.Pack`](https://www.nuget.org/packages/NuGet.Build.Tasks.Pack) - NuGet tasks for MSBuild and dotnet pack.
+* [`NuGet.Build.Tasks`](https://www.nuget.org/packages/NuGet.Build.Tasks) - NuGet tasks for MSBuild and dotnet restore.
+* [`NuGet.CommandLine.XPlat`](https://www.nuget.org/packages/NuGet.CommandLine.XPlat) - NuGet executable wrapper for the dotnet CLI nuget functionality.
```

Comment:
These are not library packages, they do not provide APIs for projects to use in their apps. With the expect of the pack tasks, they're only packages due to the way that the .NET SDK is composed.